students/ahwall/Lesson 02 Homework/FizzBuzz Exercise_HW2/FizzBuzz.py

```diff
@@ -0,0 +1,10 @@
+def FizzBuzz():
+    for i in range(1, 101):
+        if i % 3 == 0 and i % 5 == 0:
```

Comment:
Nice fizzbuzz! Check out the one-liner in the solutions if you have time and are interested in how to do this in one line!